packages/sdk/src/build/build-params.js

```diff
@@ -1,9 +0,0 @@
-import {pipe, makeParam} from "../interaction/interaction.js"
-
-export function params(px = []) {
-  return pipe(px.map(makeParam))
```

Comment:
In other areas of the SDK, when deprecating functionality the pattern we commonly use is to emit a console warning when deprecated functionality is being used, and link out to a transition guide so the user can change their code to use non-deprecated functionality.

https://github.com/onflow/flow-js-sdk/blob/9cf62bfaabb02444e6daa24b1ee10faeed40f732/packages/sdk/src/build/build-get-block-by-height.js#L5-L16

Reply:
What about the exports from?

Reply:
Transition doc is here: https://github.com/onflow/flow-js-sdk/blob/master/packages/sdk/TRANSITIONS.md#0001-deprecate-params

And in this case is should be an error, and state it has been removed i am thinking

Reply:
Updated to export deprecation.error from [/utils](https://github.com/onflow/flow-js-sdk/blob/07869d2b0236062070ee9f5586ad617f22648211/packages/sdk/src/utils/index.js)